ui/app/src/elements/components/table.ts

```diff
@@ -73,7 +73,7 @@ export class StatefulTable extends NHComponentShoelace {
         heading: generateHeaderHTML('Resource', resourceName),
         decorator: (resource: any) => html`<div
           style="width: 100%; display: grid;place-content: start center; height: 100%; justify-items: center;"
-          ${typeof resource.eh[1] === 'function' ? ref((e) => resource.eh[1](e as HTMLElement, resource.eh[0])) : null}
+          ${typeof resource.eh[1] === 'function' ? ref((e) => e ? resource.eh[1](e as HTMLElement, resource.eh[0]): null) : null}
```

Comment:
This was added because some how e was undefined in some render cycles which prevented the widgets from rendering. Not sure what caused this, but the ternary operator fixes the issue.